src/components/util/search-filters.js

```diff
@@ -246,6 +355,7 @@ export function getDateHistogramIntervalFromFacet(facet){
 
 
 /**
+ * @deprecated
```

Comment:
Might be safe to delete the deprecated functions. Maybe if have time can double-check via Search that these functions are not used anymore in CGAP, 4DN, and SPC, and then delete + push to master here.

Reply:
Sure! we'll check once more and delete if not referenced anymore.